src/SprykerEco/Zed/Computop/Business/Order/Mapper/PostPlace/PayuCeeSingleMapper.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Computop\Business\Order\Mapper\PostPlace;
+
+use Generated\Shared\Transfer\ComputopPayuCeeSingleInitResponseTransfer;
+use Generated\Shared\Transfer\ComputopPayuCeeSinglePaymentTransfer;
+use Generated\Shared\Transfer\PaymentTransfer;
+use SprykerEco\Shared\Computop\ComputopConfig;
+use SprykerEco\Zed\Computop\Business\Order\Mapper\MapperInterface;
+
+class PayuCeeSingleMapper implements MapperInterface
+{
+    /**
+     * @return string
+     */
+    public function getMethodName(): string
+    {
+        return ComputopConfig::PAYMENT_METHOD_PAYU_CEE_SINGLE;
+    }
+
+    /**
+     * @param \Generated\Shared\Transfer\PaymentTransfer $paymentTransfer
+     *
+     * @return \Generated\Shared\Transfer\ComputopPayuCeeSinglePaymentTransfer|null
+     */
+    public function getComputopTransfer(PaymentTransfer $paymentTransfer): ?ComputopPayuCeeSinglePaymentTransfer
```

Comment:
Method called `getComputopTransfer ` but returns `ComputopPayuCeeSinglePaymentTransfer `

Reply:
I can't rename this method. The class implement MapperInterface.